arch/arm/src/armv7-m/arm_vectors.c

```diff
@@ -51,6 +52,45 @@
 #  error ARMV7M_PERIPHERAL_INTERRUPTS must be defined to the number of I/O interrupts to be supported
 #endif
 
+/* Vector Table Offset Register (VECTAB).  This mask seems to vary among
```

Comment:
let's move these macro to ram_vectors.h too

Reply:
@xiaoxiang781216 
I have done !

Reply:
but no change?

Reply:
@xiaoxiang781216 
But I got the e-mail : Run cancelled...

Reply:
@xiaoxiang781216 
> but no change?

I moved "#ifndef ARMV7M_PERIPHERAL_INTERRUPTS" to ram_vector.h
Your intention is also to move "#ifndef NVIC_VECTAB_TBLOFF_MASK" and "#define VECTOR_ALIGN" ?

Reply:
yes, all macro could be moved to header file

Reply:
OK, I will try it later, please give me some time.

Reply:
@xiaoxiang781216 
I have done ! (sorry, I fogot this comment, and thank you for your advice!)